code/modules/research/xenoarchaeology/finds/finds_misc.dm

```diff
@@ -25,7 +25,6 @@
 	return
 
 /obj/item/weapon/shard/plasma/attackby(obj/item/weapon/W as obj, mob/user as mob, params)
-	..()
 	if ( istype(W, /obj/item/weapon/weldingtool))
```

Comment:
I'm going to go through the codebase and add semicolons and {} to the appropriate areas in each statement now.